expression/builtin_string_vec_test.go

```diff
@@ -34,7 +34,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Unhex: {
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randHexStrGener{10, 100}}},
 	},
-	ast.Trim:      {},
+	ast.Trim: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETString}},
```

Comment:
We may need to specify the data generators to test the cases when `len(str) < len(remstr)` or `len(str) >= len(remstr)`.

Reply:
I use randLenStrGener to generate different length case